sycl/test-e2e/Plugin/interop-level-zero-buffer-ownership.cpp

```diff
@@ -17,10 +17,13 @@
 
 // Keep ownership
 // CHECK: zeMemFree
+// CHECK: zeMemFree
 
 // Transfer ownership
 // CHECK: zeMemFree
 // CHECK: zeMemFree
+// CHECK: zeMemFree
+// CHECK: zeMemFree
```

Comment:
There is `CHECK-COUNT-4:` (or any other number) that can be used here.

Reply:
thanks @aelovikov-intel . I will use that.